### ISSUE TITLE
Where are the filters for On Premise TFS 2015/7/8 ?

### ISSUE THREAD

**Issue by StingyJack** [MicrosoftDocs/vsts-docs#2942]
The  marketplace only lists Azure DevOps, and I'm not interested in extensions that only target the hosted platform. TFS 2015 is years away from EOL and is still supported. I expect to be able to find things for it on the marketplace.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 145e9748-0039-e0df-2bc1-7ad5e74ab75a
* Version Independent ID: cf629735-b15d-3948-86a3-6fffe25ec0ae
* Content: [Marketplace support](https://docs.microsoft.com/en-us/azure/devops/marketplace/marketplace-billing-qa?view=vsts)
* Content Source: [docs/marketplace/marketplace-billing-qa.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/marketplace/marketplace-billing-qa.md)
* Product: **devops**
* GitHub Login: @elbatk
* Microsoft Alias: **elbatk**

**Comment by elbatk** [comment]
Hi @StingyJack , I'm going to add @harishkragarwal for help here. However, this is more of a product suggestion than a docs issue so I'm going to close the issue. Harish will make better use of your feedback, or you can reach out here: General marketplace help - Contact the Visual Studio Marketplace team (vsmarketplace@microsoft.com). Thank you!

**Comment by harishkragarwal** [comment]
@StingyJack : by 'Azure DevOps' I imply both Azure DevOps Services (formerly VSTS) and Azure DevOps Server (formerly TFS). Am with you w.r.t. TFS 2015 support. If you visit https://marketplace.visualstudio.com/search?target=AzureDevOps then you will able to search based on Cloud vs On Premises. Probably in future I can consider adding TFS versions as an explicit option there.

**Comment by StingyJack** [comment]
@elbatk - thats fine, I think I clicked on the FAQ link and found my way here, but its not easy to consistently find a product team at microsoft. Thanks for the nudge.

@harishkragarwal - So when I look at something like this..
![image](https://user-images.githubusercontent.com/15381181/50809008-47952e00-12cf-11e9-85ef-94c00b61e24d.png)
Can I install it to an on-premise TFS 2015 server or not? 

The versions used to all be listed so I would know what I could install and that should work, and what I shouldn't bother trying. Now I'm probably just not going to bother trying any of them, because I don't need to sink time into something that may not even be compatible.

I'm more than a bit confounded at the loss of specificity in product support. Just because the new offering is called "Azure DevOps" doesn't retroactively change the previous product names from TFS 20XX. Windows 95 didn't become known as Windows 10 when Windows 10 was released. 

(Its not that some of us want to stay on an older platform, its just hard to find the time to upgrade - and deal with any rollback if it happens to fail - when there are a so many daily minimpediments.)
